biothings/tests/biothings_pytest_plugin/biothings_pytest_plugin.py

```diff
@@ -0,0 +1,246 @@
+import pytest
+import boto3
+import os
+import requests
+from _pytest.terminal import TerminalReporter
+
+
+"""
```

Comment:
I would move this documentation string to line 1 so that it acts like the docstring for the module since it is high quality